src/docs/teams/reference/configuration.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Teams | Configuration
+layout: docs.hbs
+---
+# Configuration
+
+## Repository Structure
```

Comment:
This part could be in the Adding Repositories section. I think that's where I'd look for it initially. However, if you planned on expanding this section in the future to contain configurable structures, then it's fine here.

Reply:
Ya the intent was for the latter scenario. It doesn't hurt to have redundant info though